pkg/util/glog/glog.go

```diff
@@ -1,101 +1,224 @@
 package glog
 
 import (
+	"bufio"
 	"fmt"
 	"io"
+	"os"
+	"runtime"
 	"strings"
+	"sync/atomic"
 
-	"github.com/golang/glog"
+	log "github.com/golang/glog"
 )
 
 // Logger is a simple interface that is roughly equivalent to glog.
 type Logger interface {
-	Is(level int) bool
-	V(level int) Logger
+	Is(level int32) bool
+	V(level int32) VerboseLogger
 	Infof(format string, args ...interface{})
+	Info(args ...interface{})
+	Warningf(format string, args ...interface{})
+	Warning(args ...interface{})
+	Errorf(format string, args ...interface{})
+	Error(args ...interface{})
+	Fatalf(format string, args ...interface{})
+	Fatal(args ...interface{})
```

Comment:
i'm guessing there's a reason @smarterclayton didn't do this when he first implemented it, so i'd like him to weigh in before we add these utility methods.